composer.json

```diff
@@ -19,6 +19,7 @@
     "require": {
         "php": "^7.4",
         "cebe/markdown": "^1.2@dev",
+        "cycle/proxy-factory": "^1.2",
```

Comment:
Should it be required directly? Do we need it all the time? If so, does it make sense to move it to yii-cycle?